lib/identity_cache/should_use_cache.rb

```diff
@@ -0,0 +1,13 @@
+module IdentityCache
+  module ShouldUseCache
+    extend ActiveSupport::Concern
+
+    included do
+      unless self.respond_to?(:should_use_cache?)
+        def self.should_use_cache?
```

Comment:
Okay, you're out-rubbying me here Dylan, how does it make a difference whether `should_use_cache` is defined in a mixin in this case?

I'll change it to what you suggested. 👍